app/assets/stylesheets/select2_overrides.scss

```diff
@@ -1,6 +1,23 @@
 .select2-container span.select2-selection--multiple {
   min-height: 150px;
-  min-width: 450px;
+  width: 350px;
```

Comment:
Hardcoded `width` with `px` unit will not work as expected !
With this approach we need to have breakpoint for each type of screen which is obviously not feasible.

It will be better if you can anyhow related the width with the parent of the element

Reply:
ok let me try something ..